test/average.test.js

```diff
@@ -12,5 +12,5 @@ it('reports the average as NaN on an empty list', ()=> {
 });
 
 it('ignores NaN in the input', ()=> {
-  expect(average([1, NaN, 2])).to.be.approximately(1.5, 0.01);
+  expect(average([1, 2])).to.be.approximately(1.5, 0.01);
```

Comment:
The intent of the test is to prove that it "ignores NaN in the input" (as stated above in line #14). How does the modified `expect` statement prove that?